company_website/static/common/scripts/external-links-handler.js

```diff
@@ -1,10 +1,15 @@
+let exception = "#talk-with-adrian"
```

Comment:
Please name it differently it confuse because exception means that is somehow raised and you trying to catch it here. This is id of div or some widget. Please change it

Reply:
Done